setup.py

```diff
@@ -0,0 +1,45 @@
+import os
+import re
+from setuptools import setup, find_packages
+from setuptools.command.install_scripts import install_scripts
+
+
+class my_install_scripts(install_scripts):
+    def write_script(self, script_name, contents, mode="t", *ignored):
+        contents = re.sub("import sys",
+                          "import sys\nsys.path.append('/opt/graphite/lib')",
+                          contents)
+        install_scripts.write_script(
```

Comment:
TypeError: must be type, not classobj.
I suspect that the top-level thing isn't really a clean class. I'll keep the thing that works that I copied from carbonate.